### ISSUE TITLE
think about this list of rules

### ISSUE THREAD

**Issue by maxkoryukov** [volebo/eslint-config-volebo#3]
JSHint config

**Branch: `feature/jshint` please**
- [x] "bitwise": true,
- [x] "camelcase": true,
- [x] "curly": false,  /\* custom */
- [x] "eqeqeq": true,
- [x] "es3": false,
- [x] "forin": true,
- [x] "freeze": true,
- [x] "immed": true,
- [x] "indent": 1,  /\* custom */
- [x] "latedef": true,
- [x] "newcap": true,
- [x] "noarg": true,
- [x] "noempty": true,
- [x] "nonbsp": true,
- [x] "nonew": true,
- [x] "plusplus": false,  /\* custom */
- [x] "quotmark": true,
- [x] "undef": true,
- [x] "unused": true,
- [x] "strict": false, /\* enable later */
- [x] "trailing": true,
- [x] "maxparams": false,
- [x] "maxdepth": false,
- [x] "maxstatements": false,
- [x] "maxcomplexity": false,
- [x] "maxlen": false,
- [x] "asi": false,
- [x] "boss": true,  /\* custom */
- [x] "debug": false,
- [x] "eqnull": true,  /\* custom */
- [x] "esnext": false,
- [x] "evil": false,
- [x] "expr": false,
- [x] "funcscope": false,
- [x] "gcl": false,
- [x] "globalstrict": false,
- [x] "iterator": false,
- [x] "lastsemic": false,
- [x] "laxbreak": false,
- [x] "laxcomma": false,
- [x] "loopfunc": false,
- [x] "moz": false,
- [x] "multistr": false,
- [x] "notypeof": false,
- [x] "proto": false,
- [x] "scripturl": false,
- [x] "smarttabs": false,
- [x] "shadow": false,
- [x] "sub": false,
- [x] "supernew": false,
- [x] "noyield": false,
- [x] "browser": true,  /\* custom */
- [x] "onevar": true  /\* custom */

``` javascript
{
  "bitwise": true,
  "camelcase": true,
  "curly": false,  /* custom */
  "eqeqeq": true,
  "es3": false,
  "forin": true,
  "freeze": true,
  "immed": true,
  "indent": 1,  /* custom */
  "latedef": true,
  "newcap": true,
  "noarg": true,
  "noempty": true,
  "nonbsp": true,
  "nonew": true,
  "plusplus": false,  /* custom */
  "quotmark": true,
  "undef": true,
  "unused": true,
  "strict": false, /* enable later */
  "trailing": true,
  "maxparams": false,
  "maxdepth": false,
  "maxstatements": false,
  "maxcomplexity": false,
  "maxlen": false,
  "asi": false,
  "boss": true,  /* custom */
  "debug": false,
  "eqnull": true,  /* custom */
  "esnext": false,
  "evil": false,
  "expr": false,
  "funcscope": false,
  "gcl": false,
  "globalstrict": false,
  "iterator": false,
  "lastsemic": false,
  "laxbreak": false,
  "laxcomma": false,
  "loopfunc": false,
  "moz": false,
  "multistr": false,
  "notypeof": false,
  "proto": false,
  "scripturl": false,
  "smarttabs": false,
  "shadow": false,
  "sub": false,
  "supernew": false,
  "noyield": false,
  "browser": true,  /* custom */
  "onevar": true  /* custom */
}
```

**Comment by maxkoryukov** [comment]
JSHINT: http://jshint.com/docs/options/

**Comment by maxkoryukov** [comment]
partially done, stopped on `"noempty": true,`

**Comment by maxkoryukov** [comment]
`feature/jshint`

**Comment by maxkoryukov** [comment]
Thanks to @jrajav for fixes in jrajav/jshint2eslint#3

**Comment by maxkoryukov** [comment]
Now it should go faster.. Or.. May be not. Need to think about each rule, so no automation, just sanity and thinking
